tests/manager/manager-debug-001.phpt

```diff
@@ -10,16 +10,18 @@ require_once __DIR__ . "/../utils/basic.inc";
 $name = tempnam(sys_get_temp_dir(), "PHONGO");
 unlink($name);
 mkdir($name);
-$manager = new MongoDB\Driver\Manager(STANDALONE, array(), array("debug" => $name));
+ini_set('mongodb.debug', $name);
+$manager = new MongoDB\Driver\Manager(STANDALONE);
 $bulk = new MongoDB\Driver\BulkWrite();
 $bulk->insert(array('_id' => 1, 'x' => 1));
 $result = $manager->executeBulkWrite(NS, $bulk);
+ini_set('mongodb.debug', 'off');
 foreach(glob($name."/*") as $file);
 $content = file($file);
 unlink($file);
 rmdir($name);
 
-echo $content[0];
+echo $content[0], $content[1];
```

Comment:
Previously, the `debug` option was applied after URI parsing, which mean that the first debugging output would have been the "Creating Manager" line. Since we need to alter the INI setting before invoking the constructor, the first line is now "Connection string". I've elected to output both for the test.